projekt-jjit.py

```diff
@@ -0,0 +1,158 @@
20%24koszty_autorskie_procent%3A%20Int%2C%20%24ppk%3A%20Int%2C%20%24ppk_pracownik%3A%20Float%2C%20%24ppk_pracodawca%3A%20Float%2C%20%24pit_26%3A%20Int%2C%20%24zwiekszone_koszty_uzyskania%3A%20Int%2C%20%24poza_miejscem_zamieszkania%3A%20Int%2C%20%24uwzglednij_kwote_wolna%3A%20Int%2C%20%24wspolne_rozliczanie%3A%20Int)%20%7B%0A%20%20calculated%3A%20salary_calc(rok_podatkowy%3A%20%24rok_podatkowy%2C%20typ_kalkulatora%3A%20%24typ_kalkulatora%2C%20typ_wynagrodzenia%3A%20%24typ_wynagrodzenia%2C%20pensja%3A%20%24pensja%2C%20pensja_miesiace%3A%20%24pensja_miesiace%2C%20koszty_autorskie%3A%20%24koszty_autorskie%2C%20koszty_autorskie_procent%3A%20%24koszty_autorskie_procent%2C%20ppk%3A%20%24ppk%2C%20ppk_pracownik%3A%20%24ppk_pracownik%2C%20ppk_pracodawca%3A%20%24ppk_pracodawca%2C%20pit_26%3A%20%24pit_26%2C%20zwiekszone_koszty_uzyskania%3A%20%24zwiekszone_koszty_uzyskania%2C%20poza_miejscem_zamieszkania%3A%20%24poza_miejscem_zamieszkania%2C%20uwzglednij_kwote_wolna%3A%20%24uwzglednij_kwote_wolna%2C%20wspolne_rozliczanie%3A%20%24wspolne_rozliczanie)%20%7B%0A%20%20%20%20miesiace%20%7B%0A%20%20%20%20%20%20miesiac%0A%20%20%20%20%20%20koszt_uzyskania%0A%20%20%20%20%20%20zaliczka%0A%20%20%20%20%20%20zdrowotne%0A%20%20%20%20%20%20chorobowe%0A%20%20%20%20%20%20rentowe%0A%20%20%20%20%20%20emerytalne%0A%20%20%20%20%20%20rentowe_pracodawca%0A%20%20%20%20%20%20emerytalne_pracodawca%0A%20%20%20%20%20%20brutto%0A%20%20%20%20%20%20netto%0A%20%20%20%20%20%20stawka%0A%20%20%20%20%20%20zaliczka%0A%20%20%20%20%20%20koszt_pracodawcy%0A%20%20%20%20%20%20wypadkowe_pracodawca%0A%20%20%20%20%20%20fundusz_pracy_pracodawca%0A%20%20%20%20%20%20fgsp_pracodawca%0A%20%20%20%20%20%20__typename%0A%20%20%20%20%7D%0A%20%20%20%20koszt_uzyskania%0A%20%20%20%20zaliczka%0A%20%20%20%20zdrowotne%0A%20%20%20%20chorobowe%0A%20%20%20%20rentowe%0A%20%20%20%20emerytalne%0A%20%20%20%20rentowe_pracodawca%0A%20%20%20%20emerytalne_pracodawca%0A%20%20%20%20zaliczka%0A%20%20%20%20netto%0A%20%20%20%20brutto%0A%20%20%20%20niedoplata%0A%20%20%20%20koszt_pracodawcy%0A%20%20%20%20fgsp_pracodawca%0A%20%20%20%20wypadkowe_pracodawca%0A%20%20%20%20fundusz_pracy_pracodawca%0A%20%20%20%20zlecenie_netto_miesiac%0A%20%20%20%20zlecenie_brutto_miesiac%0A%20%20%20%20zlecenie_pracodawca_miesiac%0A%20%20%20%20dzielo_netto_miesiac%0A%20%20%20%20dzielo_brutto_miesiac%0A%20%20%20%20dzielo_pracodawca_miesiac%0A%20%20%20%20praca_netto_miesiac%0A%20%20%20%20praca_brutto_miesiac%0A%20%20%20%20praca_pracodawca_miesiac%0A%20%20%20%20__typename%0A%20%20%7D%0A%7D%0A&operationName=sg_money_gielda_kalulator_wynagrodzen&variables=%7B%22rok_podatkowy%22%3A2022%2C%22pensja%22%3A{}%2C%22typ_kalkulatora%22%3A%22UMOWA_O_PRACE%22%2C%22typ_wynagrodzenia%22%3A%22brutto%22%2C%22koszty_autorskie%22%3A0%2C%22koszty_autorskie_procent%22%3A0%2C%22poza_miejscem_zamieszkania%22%3A0%2C%22wspolne_rozliczanie%22%3A0%2C%22uwzglednij_kwote_wolna%22%3A1%2C%22ppk%22%3A0%2C%22ppk_pracownik%22%3A2%2C%22ppk_pracodawca%22%3A1.5%2C%22pit_26%22%3A0%7D".format(salary)
```

Comment:
PEP8 mowi o 80 znakach w linii. Nowsze podejście mówi o 88-ish. Tak czy inaczej fajnie byłoby nie miec az tak dlugich linii. Możnaby ładnie sformatować to zapytanie grapgql i wrzucic je na góre pliku na stała.